### PR TITLE
To Do List First part

### PR DESCRIPTION
**Pull request by Zia123456**
- Best practice for HTML and CSS is used
- Best practice for JavaScript is used
- The project set up with webpack
- CSS is loaded by Webpack Style/CSS loader
- An array of objects created to store a to-do list
- A function created to iterate over the tasks array
- List of task created dynamically

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,35 @@
...
+
+## Built With
+- html
+- css
+- js
+## Live Demo (if available)
+Live Demo [Link](#)
+## Getting Started
+Simply you can open index.html file with any browser and, you will see the home page.
+for opennig index.html go to to-do-list directory, find index.html, right click on it, click on open with and, click on any browser which is available on your pc/labtop.
```
Comment (ElsonOtake) [inline README.md:12]: - [OPTIONAL] I think it's better to explain how someone who wants to check your application all the steps to execute your code. How to clone your repository, run `npm install` and `npm start`. The recruiters and hiring managers can check your work easily.

File: src/index.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+// array of to do object
+const list = [
+  { description: 'was the dishes', completed: false, index: 1 },
+  { description: 'was the dishes', completed: false, index: 2 },
+  { description: 'was the dishes', completed: false, index: 3 },
+];
+
+// function to display to do list
+const displayList = function display() {
```
Comment (ElsonOtake) [inline src/index.js:16]: - [x] Please sort your array by index before displaying the list, as required on the project. Your code is displaying the list without considering the index field value.

**Review by ElsonOtake — CHANGES_REQUESTED** [review]
STATUS: CHANGES REQUIRED 🟡 

Hi @Zia123456,

Good job so far! 🎉 🎉  I love your to do list!! Well done!
There are some issues that you still need to work on to go to the next project but you are almost there! 🥲 🥲 

woohoo-youre-almost

## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️
- Professional README file. ✔️
- PR has a good title and summary. ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Felix45 — APPROVED** [review]
# Approved :green_circle: 
Hi @Zia123456 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights :heavy_check_mark: 
- [x] Implemented all required changes from the previous reviewer :white_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**